ios/FluentUI/Vnext/Button/ButtonTokens.swift

```diff
@@ -30,145 +30,291 @@ import UIKit
+                hover: aliasTokens.backgroundColors[.brandHover],
+                pressed: aliasTokens.backgroundColors[.brandPressed],
+                selected: aliasTokens.backgroundColors[.brandSelected],
+                disabled: aliasTokens.backgroundColors[.brandDisabled]
+            )
+        case .ghost:
+            return .init(
+                rest: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0)),
+                hover: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0)),
+                pressed: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0)),
+                selected: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0)),
+                disabled: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0))
+            )
+        default:
+            return .init(
+                rest: aliasTokens.backgroundColors[.brandRest],
+                hover: aliasTokens.backgroundColors[.brandHover],
+                pressed: aliasTokens.backgroundColors[.brandPressed],
+                selected: aliasTokens.backgroundColors[.brandSelected],
+                disabled: aliasTokens.backgroundColors[.brandDisabled]
+            )}
+    }()
+
+    lazy var backgroundColor: ButtonDynamicColors = {
+        switch style {
+        case .primary:
+            return .init(
+                rest: aliasTokens.backgroundColors[.brandRest],
+                hover: aliasTokens.backgroundColors[.brandHover],
+                pressed: aliasTokens.backgroundColors[.brandPressed],
+                selected: aliasTokens.backgroundColors[.brandSelected],
+                disabled: aliasTokens.backgroundColors[.brandDisabled]
+            )
+        case .accentFloating:
+            return .init(
+                rest: aliasTokens.backgroundColors[.brandRest],
+                hover: aliasTokens.backgroundColors[.brandHover],
+                pressed: aliasTokens.backgroundColors[.brandPressed],
+                selected: aliasTokens.backgroundColors[.brandSelected],
+                disabled: aliasTokens.backgroundColors[.brandDisabled]
+            )
+        case .subtleFloating:
+            return .init(
+                rest: aliasTokens.backgroundColors[.neutral1],
+                hover: aliasTokens.backgroundColors[.neutral1],
+                pressed: aliasTokens.backgroundColors[.neutral5],
+                selected: aliasTokens.backgroundColors[.neutral1],
+                disabled: aliasTokens.backgroundColors[.neutral1]
+            )
+        default:
+            return .init(
+                rest: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0)),
+                hover: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0)),
+                pressed: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0)),
+                selected: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0)),
+                disabled: DynamicColor(light: ColorValue(r: 0, g: 0, b: 0, a: 0))
```

Comment:
Do we not have a clear option?

Reply:
Not yet, but you're right that this is repeated too many times. I'll add one.